doc/api/addons.markdown

```diff
@@ -479,15 +531,14 @@ Test it with:
 
 ### Factory of wrapped objects
 
-This is useful when you want to be able to create native objects without
-explicitly instantiating them with the `new` operator in JavaScript. For
-example:
+Alternatively, it is possible to use a factory pattern to avoid explicitly
+creating object instances using the JavaScript `new` operator:
 
     var obj = addon.createObject();
     // instead of:
     // var obj = new addon.Object();
 
-Let's register our `createObject` method in `addon.cc`:
+First, the `createObject` method is implemented in `addon.cc`:
```

Comment:
methods should have `()` with them for consistency with the rest of the docs I believe, there's probably some other places in this doc that needs to be fixed